test/python/test_job_processor.py

```diff
@@ -157,12 +157,18 @@ def test_init_job_processor(self):
             job_list.append(qjob)
```

Comment:
qjob, qobj and job are very similar words and could lead to confusion. I'd recommend use complete words separated by underscores like PEP8 suggest (except for qobj): "q_job" or even better "quantum_job" are good candidates.

Reply:
I changed some of these but there are probably more. WIP.